client/src/meta/env.meta.js

```diff
@@ -0,0 +1,6 @@
+const meta = require('./meta-util')
```

Comment:
this is in the `client/src/` directory, why no `import` nor `export`? if they're going to be run using node, I wouldn't put the inside this directory but some `client/scripts`, maybe.

Reply:
The metaprogramming `.meta.js` using `val-loader` don't work with Babel currently and are rare enough that I don't think it's worth spending too much time trying to make it work.

**TODO:** Add this limitation to `docs/metaprogramming.md`